internal/backends/winit/renderer/sw.rs

```diff
@@ -19,8 +20,13 @@ impl<const BUFFER_COUNT: usize> super::WinitCompatibleRenderer for SoftwareRende
         SoftwareRenderer::new(window_adapter_weak.clone())
     }
 
-    fn create_canvas(&self, window_builder: winit::window::WindowBuilder) -> Self::Canvas {
-        let (opengl_context, window) = crate::OpenGLContext::new_context(window_builder);
+    fn create_canvas(
+        &self,
+        window: &dyn raw_window_handle::HasRawWindowHandle,
+        display: &dyn raw_window_handle::HasRawDisplayHandle,
+        size: PhysicalWindowSize,
+    ) -> Self::Canvas {
+        let opengl_context = crate::OpenGLContext::new_context(window, display, size);
```

Comment:
Unrelated, but is there still no way to create a software context?

Reply:
Do you mean a software OpenGL context?

Reply:
I mean something onto which the software renderer could render without requiring femtovg

Reply:
Ah no, this remains a missing piece / missing crate - unfortunately. That said, I think for improved compatibility on Windows I think it would make sense to switch that over to Skia.